Update autolaunch-store-options.md (#5001)
OfficeDev/office-js-docs-pr: docs/outlook/autolaunch-store-options.md

## Deploy add-in with flight code

> After the flight codes are set, you'll receive the URL from the app certification team. You can then share the URL with admins privately. Please note that it may take up to 24 hours after certification of your app for the flight codes to take effect and for AppSource to hide your existing listing from search results.

Reviewer: ```suggestion
After the flight codes are set, you'll receive the URL from the app certification team. You can then share the URL with admins privately.

> [!NOTE]
> Once your add-in is certified, it may take up to 24 hours for the flight codes to take effect and for AppSource to hide your existing listing from the search results.
```
